### DISCUSSION TITLE
Account has been suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#118533] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My Felix4080 ([github.com/felix4080](http://github.com/felix4080)) account was suspended, I did not receive anything by gmail. What do I need to do to restore it? I created a support ticket on the site 4 days ago, but they still haven’t responded to me. Please help me!

**Comment by devashish2024** [comment]
Account suspensions are only on illegal activities. What you did?

edit: visit other discussions, they were not reactivated...?

**Comment by albinoguip** [comment]
Hi! Did you managed to reactivate your account? the same happened to me

**Comment by ebndev** [comment]
Hi [felix-4080](https://github.com/felix-4080),

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

We appreciate your understanding!
